Formula/opencascade.rb

```diff
@@ -0,0 +1,45 @@
+class Opencascade < Formula
+  desc "3D modeling and numerical simulation software for CAD/CAM/CAE"
+  homepage "https://www.opencascade.com/content/core-technology"
+  url "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/tags/V7_2_0;sf=tgz"
+  version "7.2.0"
+  sha256 "adb6fc44efebdb09e30a487fa412afae9599992dc771b0039b4ebf306891a558"
+  head "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=HEAD;sf=tgz"
+
+  depends_on "cmake" => :build
+  depends_on "freetype"
+  depends_on "freeimage"
+  depends_on "tbb"
+  depends_on "gl2ps"
+  depends_on "tcl-tk"
+  depends_on "doxygen"
+
+  def install
+    args = std_cmake_args
+    args << "-DUSE_GL2PS:BOOL=ON"
+    args << "-DUSE_FREEIMAGE:BOOL=ON"
+    args << "-DUSE_TBB:BOOL=ON"
+    args << "-DINSTALL_DOC_Overview:BOOL=ON"
+
+    mkdir "build" do
```

Comment:
does an in-tree build work?

Reply:
Opencascade recommends and out-of-souce build.  An earlier version of the legacy formula did an in-tree build that resulted in in all the intermediate binaries being included in the bottle.

Reply:
ok but does that still happen?

Reply:
Don't know - that is from recollection circa Opencasde 6.x.  I can regress to see but what is the advantage of an in-tree build?

Reply:
just two fewer lines. Fewer lines = betterer

Reply:
LoL - I will see if I can do betterer ;)

Reply:
Outside source-tree build:
```
==> Not running post_install as we're building a bottle
You can run it manually using `brew postinstall opencascade`
==> Summary
🍺  /usr/local/Cellar/opencascade/7.2.0: 9,235 files, 164.6MB, built in 14 minutes 3 seconds
[blacey@bbl /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula (master)]$ brew bottle opencascade
==> Determining opencascade bottle rebuild...
==> Bottling opencascade-7.2.0.high_sierra.bottle.1.tar.gz...
==> Detecting if opencascade-7.2.0.high_sierra.bottle.1.tar.gz is relocatable... 
[blacey@bbl /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula (master)]$ ls -lh opencascade-7.2.0.high_sierra.bottle.1.tar.gz                    
[blacey@bbl /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula (master)]$ ls -lh opencascade-7.2.0.high_sierra.bottle.1.tar.gz 
-rw-r--r--  1 blacey  admin    50M Aug 31 05:47 opencascade-7.2.0.high_sierra.bottle.1.tar.gz
```

In source-tree build:
```
==> Not running post_install as we're building a bottle
You can run it manually using `brew postinstall opencascade`
==> Summary
🍺  /usr/local/Cellar/opencascade/7.2.0: 9,235 files, 164.6MB, built in 14 minutes 35 seconds
[blacey@bbl /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula (master *)]$ brew bottle opencascade
==> Determining opencascade bottle rebuild...
==> Bottling opencascade-7.2.0.high_sierra.bottle.1.tar.gz...
==> Detecting if opencascade-7.2.0.high_sierra.bottle.1.tar.gz is relocatable... 
[blacey@bbl /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula (master *)]$ ls -lh opencascade-7.2.0.high_sierra.bottle.1.tar.gz
-rw-r--r--  1 blacey  admin    50M Aug 31 05:47 opencascade-7.2.0.high_sierra.bottle.1.tar.gz
```

Bottom-line, looks like an *in source-tree* build yields the same results so I will change the formula to be *betterer* ;)